### ISSUE TITLE
High read abundance but no bins

### ISSUE THREAD

**Issue by zahidrehman** [BinPro/CONCOCT#211]
Dear, I have metagenome assembly of two samples that I am trying to bin. Although I have a high abundance of Proteobacterial reads, after binning I don't get any Proteobacterial bin. What could be the possible reasons? Coverage should not be an issue; since I have high read abundance. I don't expect too much strain variation since these are biofilm samples. I know CONCOCT struggles with binning if samples are too few. However, I do get bins for other bacteria whose relative abundance of reads is less than Proteobacteria. Any comments would be highly appreciated.
Best Regards
Zahid

**Comment by zahidrehman** [comment]
Dear Andres,
I made the assembly with two samples (which were actually duplicates of one sample) sequenced by Illumina HiSeq 4000. In total, I had 10 samples but the assembler (metaSpades) failed to make a single assembly for all samples. So I had to assemble individual samples (duplicates). I was reading that the efficiency of CONCOCT decreases below 50 samples. Usually, in metagenomics one don't have that many samples and also sometimes it is not possible to make a single assembly of a large number of samples, like in this case. Is there a way around this limitation of the number of samples? 
Best Regards
Zahid
